bin/proxy.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+require('babel-register');
+
+const http = require('http');
```

Comment:
I tried it and it doesn't work. That's surprising to me though. I thought `require('babel-register')` was magic enough to make it work.